src/atoms/OverlayingImageLabel.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import { Image } from './Image';
+
+const OverlayingImageLabel = styled(Image)`
```

Comment:
```suggestion
const OverlayingImage = styled(Image)`
```
how do you think

Reply:
I like Alex's naming better, it tells me that this is a label that happens to be an image